### PR TITLE
fix: help header alignment

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
fix

**Did you add tests for your changes?**
nope
**If relevant, did you update the documentation?**
nope
**Summary**

help header alignment was a bit off. 
## Before
Screenshot at 2020-12-08 16-54-32

## After
Screenshot at 2020-12-08 17-01-45

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

**Comment by alexander-akait** [comment]
Now it looks on small console? less 80 characters?

File: packages/webpack-cli/lib/groups/runHelp.js [hunk 1]
```diff
@@ -112,10 +112,10 @@ const outputHelp = (args) => {
 
         const title = bold('⬡                     ') + underline('webpack') + bold('                     ⬡');
```
Comment (rishabh3112) [inline packages/webpack-cli/lib/groups/runHelp.js:113]: Convert to tabs as well
```suggestion
        const title = bold('⬡\t\t\t\t\t') + underline('webpack') + bold('\t\t\t\t\t⬡');
```

**Comment by snitin315** [comment]
> Now it looks on small console? less 80 characters?

@alexander-akait can you clarify?

**Comment by alexander-akait** [comment]
open terminal do very small width on opened window and run `webpack --help`, it should be readable

**Comment by snitin315** [comment]
> open terminal do very small width on opened window and run `webpack --help`, it should be readable

Hmm, then we should align everything to `left`. including `docs link` & `⬡                       webpack                         ⬡`

**Comment by alexander-akait** [comment]
Yes,or we can use https://github.com/Automattic/cli-table, but need investigate

File: packages/webpack-cli/lib/groups/runHelp.js (hunk 1 above)
Comment (snitin315) [reply packages/webpack-cli/lib/groups/runHelp.js:113]: some issues with using `tabs` here. Don't know why but indentation is not equal on both sides if using tabs.

**Comment by snitin315** [comment]
Let's keep it simple and align text to left. I'm afraid using  https://github.com/Automattic/cli-table for help ouput will require a lot of refactor.

**Comment by snitin315** [comment]
Screenshot at 2020-12-08 18-09-50

**Comment by alexander-akait** [comment]
Because I say again, we need refactor, no more merging before refactoring. we should migrate on `help` callback (commander API)

**Comment by alexander-akait** [comment]
Help was refactored, so we don't need this anymore, commander do it automatically, but without colors, we will improves colors in future, it is not high priority, also our help was broken in many cases

**Comment by alexander-akait** [comment]
Anyway thanks

**Pull request closed without merging**